### ISSUE TITLE
Can't understand error

### ISSUE THREAD

**Issue by HiHoS** [jslint-org/jslint#372]
I have JS with:

let SectObj = [
   {fstSlide:27, lastSlide:28, totSlides:2, desc:"3. Aviation"}
];

It gives an error:

Expected property 'desc' to be ordered before property 'totSlides'.
   {fstSlide:27, lastSlide:28, totSlides:2, desc:"3. Aviation"}

If I write as:

let SectObj = [
   {desc:"3. Aviation", fstSlide:27, lastSlide:28, totSlides:2}
];

It does not give error.  I don't understand what the problem
with the 1st iteration is?

**Comment by kaizhu256** [comment]
jslint is making a style-complaint that the properties are not sorted.  the reasoning is to prevent chaotic/haphazard property listings for large dictionaries/configs/options.  you can disable the style-warning with directive `/*jslint unordered*/`

```js
// properties are unsorted
{
    fstSlide:27,
    lastSlide:28,
    totSlides:2,
    desc:"3. Aviation" // Expected property 'desc' to be ordered before property 'totSlides'
}

// properties sorted in ascii-order
{
    desc:"3. Aviation",
    fstSlide:27,
    lastSlide:28,
    totSlides:2
}
```
